collections/river-and-lake-ice-extent-s2.yaml

```diff
@@ -0,0 +1,74 @@
+Name: Sentinel 2 River and Lake Ice Extent
```

Comment:
I think the title format for `River and Lake Ice Extent S1+S2 ` looks good to avoid confusing with S2

something like: `River and Lake Ice Extent - Sentinel 2`